examples/jsm/renderers/webgpu/WebGPUTextures.js

```diff
@@ -184,6 +185,23 @@ class WebGPUTextures {
 
 			renderTargetProperties.colorTextureGPU = colorTextureGPU;
 
+			if ( this.sampleCount > 1 ) {
+
+				const colorBuffer = device.createTexture({
+					size: {
+						width: width,
+						height: height,
+						depth: 1
+					},
+					sampleCount: this.sampleCount,
+					format: GPUTextureFormat.BRGA8Unorm,
+					usage: GPUTextureUsage.OUTPUT_ATTACHMENT
+				});
+
+				renderTargetProperties.colorBuffer = colorBuffer;
+
+			}
+
```

Comment:
I'm not sure about this bit. The render target already has a configured color buffer. 

I suggest we look at multisampled render targets in another PR and thus revert the changes to `WebGPUTextures` for now.

Reply:
[WebGPURenderPipelines creates render pipeline with parameters.sampleCount with this change.](https://github.com/mrdoob/three.js/pull/20281/files#diff-b2e471696b2cc2a9842ff64fe2ec36bbR166) Attachment seems to need to have the same number sampleCount but I want `renderTarget.texture` to be non-multisampled texture because `WebGPURenderer` doesn't support multisampled-texture as input yet. So `.textureGPU` for resolveTarget (output, non-multisampled) and `.colorBuffer` for attachment (multisampled).

But I'm not sure if this is the best approach and I want to revisit later. So agreed with reverting for now. I added console error so far for render target + antialiasing.